### PR TITLE
fix #185; allow multiple users in the same python script

### LINKED ISSUE #185: Will pyral support using multiple api keys for multiple users.
I have a program that has two API keys. 
I create two separate Rally objects with these two keys,
```
rally1 =  Rally(server=server, apikey=apikey1, workspace=workspace, project=project, headers={'version': 'v2.0'})
rally2 = Rally(server=server, apikey=apikey2, workspace=workspace, project=project, headers={'version': 'v2.0'})
```
It seems that rally1 is not valid and only rally2 is valid. Is it possible that `Rally` has some global data that is overwritten by creating a second rally object with a different apikey.

### PR DESCRIPTION
**Pull request by Celthi**
fix #185

### CODE REVIEW

**Review by jofegan — COMMENTED** [review]
Making a copy of the dict makes sense to me, but I would use dict.copy() explicitly so that the intent is clear to future readers.

File: pyral/restapi.py [hunk 1]
```diff
@@ -218,7 +218,9 @@ def __init__(self, server=SERVER, user=None, password=None, apikey=None,
 ##        print("\n requests lib in %s" % requests.__file__)
 ##
         self.session = requests.Session()
-        self.session.headers = RALLY_REST_HEADERS
+        for k, v in RALLY_REST_HEADERS.items():
+            self.session.headers[k] = v
+
```
[the remark this answers is not in the document]
Comment (Celthi) [reply pyral/restapi.py]: Thanks!. I was aware of 'copy` function. Changed. Could you review it again~

**Comment by Celthi** [comment]
> Making a copy of the dict makes sense to me, but I would use dict.copy() explicitly so that the intent is clear to future readers.

Good catch! Changed.

**Comment by Celthi** [comment]
Any plan to merge this fix?

**Comment by klehman-rally** [comment]
This mod is included in the 1.6.0 release.

**Pull request closed without merging**